examples/HeatMap.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
```

Comment:
There are subtle differences between this file and the latest structure of the examples, e.g. Web World Wind versus Web WorldWind and the declaration of the canvas. Please review it thoroughly. You can use https://github.com/NASAWorldWind/WebWorldWind/blob/develop/examples/GeoJSON.html as reference.